resources/type-system/inference.md

```diff
@@ -276,9 +284,14 @@ The function **futureValueTypeSchema** is defined as follows:
 - **futureValueTypeSchema**(`FutureOr<S>`) = `S`, for all `S`.
 - **futureValueTypeSchema**(`void`) = `void`.
 - **futureValueTypeSchema**(`dynamic`) = `dynamic`.
-- **futureValueTypeSchema**(`_`) = `_`.
+- **futureValueTypeSchema**(`?`) = `?`.
```

Comment:
I have an open PR which is changing all uses of `?` to `_`, so let's keep this as `_`.

Reply:
SG, done!